src/debug-configuration/subproviders/pyocd-configuration-provider.ts

```diff
@@ -39,6 +41,23 @@ export class PyocdConfigurationProvider extends BaseConfigurationProvider {
         }
     }
 
+    protected resolveCmsisPackRootPath(target: TargetConfiguration): void {
+        const environmentValue = process.env.CMSIS_PACK_ROOT;
+        if (environmentValue) {
+            return;
+        }
+
+        if(target.environment?.CMSIS_PACK_ROOT) {
+            return;
+        }
+        const cmsisPackRootDefault = os.platform() === 'win32'
+            ? path.join(process.env['LOCALAPPDATA'] ?? os.homedir(), 'arm', 'packs')
```

Comment:
nit: This will work because of case-insensitive paths on Windows. But strictly speaking it should be capital `A` and `P` in the Windows variant. See https://github.com/Open-CMSIS-Pack/cmsis-toolbox/blob/main/docs/installation.md#default-values